content/en/docs/references/_index.md

```diff
@@ -41,9 +41,8 @@ The CUE Go APIs in the main repo are organized as follows:
 - [**cue**](https://godoc.org/cuelang.org/go/cue):
   core APIs related to parsing, formatting, loading and running CUE programs.
   These packages are used by all other packages, including the command line tool.
-- [**doc**](https://godoc.org/cuelang.org/go/doc):
-  [CUE documentation](/docs/references),
-  including tutorials and the reference manual.
+- **doc**: [CUE documentation](/docs/), including [tutorials](/docs/tutorials)
```

Comment:
This is a slightly odd bullet point to my mind, because this list of bullets is meant to be talking about "The CUE Go APIs in the main repo are organized as follows:"

`doc` is not a Go package for one, so this could only be a reference to https://github.com/cuelang/cue/tree/master/doc. I'd be tempted to drop this bullet altogether.

Reply:
I agree. Dropped the line since it is misplaced here.